content/en/blog/2023-05-01-april-update/index.md

```diff
@@ -0,0 +1,255 @@
+---
+author: dholbach
+date: 2023-04-11 06:30:00+00:00
+title: April 2023 Update
+description: "XX"
+url: /blog/2023/XX/april-2023-update/
+tags: [monthly-update]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+<!--
+
+Have a look at these documents
+
+- internal_docs/how-to-do-the-monthly-update.md
+  online: https://github.com/fluxcd/website/blob/main/internal_docs/how-to-do-the-monthly-update.md
+- internal_docs/how-to-write-a-blog-post.md
+  online: https://github.com/fluxcd/website/blob/main/internal_docs/how-to-write-a-blog-post.md
+
+to get more background on how to publish this blog post.
+
+-->
+
+As the Flux family of projects and its communities are growing, we
+strive to inform you each month about what has already landed, new
+possibilities which are available for integration, and where you can get
+involved. Read [our last update here](/blog/2023/04/march-2023-update/).
+
+It's the beginning of May 2023 - let's recap together what
+happened in April - it has been a lot!
+
+## News in the Flux family
+
+### Next Flux release: more stability and performance improvements
+
+### Security news
+
+### Flagger 1.30.0
+
+- <https://github.com/fluxcd/flagger/pull/1413>
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
```

Comment:
@jamwils Anything you'd like to add here for April?

Reply:
https://www.weave.works/blog/weave-gitops-release-2023-04 maybe?